MyCryptoHQ/MyCrypto
# Add Multi TXs to TX History

Steps to test:
- Go through a Multi TX flow (membership, rep migration etc)
- Refrain from refreshing until the test is done (to make sure you don't fetch the tx's from etherscan)
- Go to the dashboard
- Check whether the transactions you just made are in the tx history
